src/dictionary-sync.js

```diff
@@ -71,45 +65,23 @@ module.exports = class DictionarySync {
     let lang = normalizeLanguageCode(langCode);
     let target = path.join(this.cacheDir, `${lang}.bdic`);
 
-    let fileExists = false;
-    try {
-      if (fs.existsSync(target)) {
-        fileExists = true;
-
-        d(`Returning local copy: ${target}`);
-        let ret = await fs.readFile(target, {});
-
-        if (ret.length < 8*1024) {
-          throw new Error("File exists but is most likely bogus");
-        }
+    if (cacheOnly) return target;
 
-        return ret;
+    try {
+      if (!fs.existsSync(target)) {
+        throw new Error(`${lang} is currently not supported`);
       }
-    } catch (e) {
-      d(`Failed to read file ${target}: ${e.message}`);
-    }
+      d(`Returning local copy: ${target}`);
+      let ret = await fs.readFile(target, {});
 
-    if (fileExists) {
-      try {
-        await fs.unlink(target);
-      } catch (e) {
-        d("Can't clear out file, bailing");
-        throw e;
+      if (ret.length < 8*1024) {
+        throw new Error("File exists but is most likely bogus");
```

Comment:
in such a case, can we delete the file instead?

Reply:
This won't happen because we are manually copying those files now